cmd/draft/connect.go

```diff
@@ -59,19 +72,43 @@ func (cn *connectCmd) run(runningEnvironment string) (err error) {
 		return err
 	}
 
-	connection, err := deployedApp.Connect(client, config, containerName)
+	var ports []string
+	if len(overridePorts) == 0 {
+		if deployedApp.OverridePorts != "" {
+			// removes multiple spaces
+			s := strings.Join(strings.Fields(deployedApp.OverridePorts), " ")
+			ports = strings.Split(s, " ")
+		}
+	} else {
+		ports = overridePorts
+	}
+
+	buildID, err := cn.client.GetLatestBuildID(context.Background(), deployedApp.Name)
+	if err != nil {
+		return fmt.Errorf("cannot get latest build id: %v", err)
+	}
+
+	fmt.Fprintf(cn.out, "Connecting to your application...\n")
+	connection, err := deployedApp.Connect(client, config, buildID, targetContainer, ports)
 	if err != nil {
 		return err
 	}
 
-	// output all local ports first - easier to spot
+	if waitFlag {
+		return
+	}
+
+	var connectionMessage = "Your connection is still active. \n"
+
 	for _, cc := range connection.ContainerConnections {
 		for _, t := range cc.Tunnels {
 			err = t.ForwardPort()
 			if err != nil {
 				return err
 			}
-			fmt.Fprintf(cn.out, "Connect to %v:%v on localhost:%#v\n", cc.ContainerName, t.Remote, t.Local)
+			m := fmt.Sprintf("Connect to %v:%v on localhost:%#v\n", cc.ContainerName, t.Remote, t.Local)
+			connectionMessage += m
```

Comment:
As of Go 1.10, there is a new [strings.Builder](https://golang.org/pkg/strings/#Builder) struct that makes this process easier. Not necessarily something that needs to be done in this PR but figured I should bring it up. :)